.github/workflows/docs.yml

```diff
@@ -18,13 +17,6 @@ jobs:
         uses: actions/checkout@v3
        
       - name: Update v1-latest using rdme
-        if: ${{ github.ref_name == 'master' }}
```

Comment:
This line is likely still warranted. Without it, b/c the `workflow_dispatch` is set, it would seem to be possible for anyone to manually update the docs.

Reply:
@maxrake Is the conditional preferred to removing the `workflow_dispatch` trigger?

Reply:
Thinking more about it and talking with @kylewillmon, I believe the correct move is to remove the `workflow_dispatch` trigger.